CHANGELOG.md

```diff
@@ -1,16 +1,28 @@
 # Change Log
 
+## v8.0.1 (February 9, 2024)
+
+### Fixed
+- Chopped header for the landscape orientation ([#531](https://github.com/etn-ccis/blui-react-native-component-library/issues/531)).
```

Comment:
Can we change this to something like `The header does not render correctly in the tablet's landscape mode`